src/Modules/Internationalisation/Console/ImportLocaleCommand.php

```diff
@@ -73,15 +73,17 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         if ($importResults->getSkippedCount() > 0) {
             $formatter->comment(
                 sprintf(
-                    'Skipped %d translations because they belong to a locale that is not installed or to a different locale than specified with the --locale option.',
+                    'Skipped %d translations because they belong to a locale that is not installed
```

Comment:
Moet je hier niet met concat werken? Anders gaat die newline toch in de output komen?